core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -714,6 +725,7 @@ public double getDouble( long flags, int key )
         throw new UnsupportedOperationException("Unknown key " + key + " for double value.");
     }
 
+    @Deprecated
```

Comment:
that is indeed important :)
Would you also add this into the https://github.com/graphhopper/graphhopper/blob/master/core/files/changelog.txt and comment on the alternative?